### PR TITLE
Add task for bad edge runs

### PR DESCRIPTION
**Pull request by lukebjerring**
Renames `edge` to `bad-edge` for a range of bad revisions.

### CODE REVIEW

File: tagger/bad_edge.go [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+)
+
+type conditionUnsatisfied struct{}
+
+func (e conditionUnsatisfied) Error() string {
+	return "Condition not satisfied"
+}
+
+func condition(run *shared.TestRun) bool {
+	if run.BrowserName == "edge" {
```
Comment (Hexcles) [inline tagger/bad_edge.go]: Looks like you meant `!=`.

Comment (lukebjerring) [reply tagger/bad_edge.go]: And that right there is why we eyeball things before we run them.

File: tagger/bad_edge.go [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+	"2ed748f10d",
+	"d44bc3ed38",
+	"168de7c332",
+	"5acd3bcf66",
+	"99e577f57c",
+}
+
+var (
+	projectID = flag.String("project",
+		"wptdashboard-staging",
```
Comment (foolip) [inline tagger/bad_edge.go:38]: How about prod?

**Review by foolip — APPROVED** [review]

File: tagger/bad_edge.go (hunk 2 above)
Comment (Hexcles) [reply tagger/bad_edge.go:38]: It can be overridden by a CLI flag.

Comment (lukebjerring) [reply tagger/bad_edge.go:38]: ^ yah, i.e. `go run tagger/bad-edge.go -project=wptdashboard`, which I already have done.

Comment (foolip) [reply tagger/bad_edge.go:38]: Okay, not an oversight :)

**Comment by foolip** [comment]
There are plenty more runs that I know to be bad, maybe a script that takes a run ID and marks the run as bad would be a good idea?

**Pull request closed without merging**